### PR TITLE
Add warnings page for Madeira

### LINKED ISSUE #66: Add Warnings Page for Madeira
https://api-lb.fogos.pt/v1/madeira/warnings

https://fogos.pt/madeira/avisos

### PR DESCRIPTION
**Pull request by jbagio**
Fixes #66 
Screenshot 2019-05-30 at 10 21 15
Screenshot 2019-05-30 at 10 27 34

So, I went with a redux-based implementation for this one, instead of just copying over the existing warnings logic to a new screen. Maybe it's a bit overkill?, but seeing as this is my first venture into this project (and Flutter in general), I thought it wouldn't hurt to spend a bit more time learning the ins and outs, and getting this consistent with the rest of the codebase.
I also took the chance to update some translations (Firemen -> Firefighters).

I think it would make sense to refactor the warnings screen logic to use this approach as well, but thought I'd ask first. It should be a fairly easy change, happy to do it either in this PR or in a separate one.

As I said, this is my first foray here, so _don't_ be gentle.

### CODE REVIEW

**Comment by jbagio** [comment]
@guivazcabral as discussed in another channel, I've also refactored the existing warnings screen.

File: lib/screens/components/warnings_list.dart [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    return Column(
+      children: <Widget>[
+        new Padding(
+          padding: new EdgeInsets.only(top: 20.0),
+        ),
+        new Expanded(
+          child: new ListView.builder(
+            itemCount: warnings.length,
+            itemBuilder: (BuildContext context, int index) {
+              var warning = warnings[index];
```
Comment (guivazcabral) [inline lib/screens/components/warnings_list.dart]: change this `var` statement to declare the `Warning` type

File: lib/screens/components/warnings_list.dart [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+              return Padding(
+                padding: const EdgeInsets.only(bottom: 10.0),
+                child: ListTile(
```
Comment (guivazcabral) [inline lib/screens/components/warnings_list.dart]: i know the structure for Madeira warnings is a bit different, but would be great if the timestamp was shown on Madeira Warnings list (for the other warnings, the timestamp is the Warning title)

**Comment by jbagio** [comment]
@guivazcabral I've applied your suggested changes and also took the chance to restyle both warnings page a little bit to look a bit more like the website. Let me know what you think of it.

Screenshot 2019-06-01 at 23 43 47
Screenshot 2019-06-01 at 23 43 27

**Pull request merged**